SECURITY.md

```diff
@@ -0,0 +1,41 @@
+# Osmosis Bug Reporting and Feature Requests
+
+Osmosis uses GitHub to manage feature requests and bugs for its platform. This is done via GitHub Issues.
+
+## Feature Request
+
+For a feature request, please [create a GitHub issue](https://github.com/osmosis-labs/osmosis/issues/new). Clearly state your use case and what value it will bring to other users or developers on Osmosis.
+
+If it is something that can be handled by a parameter change, discuss it on [the discussions](https://github.com/osmosis-labs/osmosis/discussions) or on [Discord](https://discord.com/invite/osmosis) in `#governance-proposals` and consider a governance proposal.
+
+## Standard Priority Bug
+
+For a bug that is non-sensitive and/or operational in nature rather than a critical vulnerability, please add it as a GitHub issue.
+
+If it is not triaged in a couple of days, feel free to tag @p0mvn, @czarcas7ic, @mattverse, @daniel-farina, @ValarDragon
+
+## Critical Bug or Security Issue
+
+If you're here because you're trying to figure out how to notify us of a security issue, please send us a message on https://security.osmosis.zone/ or via [Telegram](https://t.me/OsmoSec) 
+
+Please avoid opening public issues on GitHub that contain information about a potential security vulnerability as this makes it difficult to reduce the impact and harm of valid security issues.
```

Comment:
Should they also be pointed to the validators security discussion chat?

Reply:
what chat are you referring to? Ideally we use this form as a filter between the reporters and wider leaks.